packages/drippie-mon/src/service.ts

```diff
@@ -0,0 +1,117 @@
+import { BaseServiceV2, Counter, validators } from '@eth-optimism/common-ts'
+import { Provider } from '@ethersproject/abstract-provider'
+import { ethers } from 'ethers'
+import * as DrippieArtifact from '@eth-optimism/contracts-periphery/artifacts/contracts/universal/drippie/Drippie.sol/Drippie.json'
+
+type DrippieMonOptions = {
+  rpc: Provider
+  drippieAddress: string
+}
+
+type DrippieMonMetrics = {
+  executableDripCount: Counter
+}
+
+type DrippieMonState = {
+  drippie: ethers.Contract
+}
+
+export class DrippieMonService extends BaseServiceV2<
+  DrippieMonOptions,
+  DrippieMonMetrics,
+  DrippieMonState
+> {
+  constructor(options?: Partial<DrippieMonOptions>) {
+    super({
+      name: 'drippie-mon',
+      loop: true,
+      loopIntervalMs: 60_000,
+      options,
+      optionsSpec: {
+        rpc: {
+          validator: validators.provider,
+          desc: 'Provider for network where Drippie is deployed',
+        },
+        drippieAddress: {
+          validator: validators.str,
+          desc: 'Address of Drippie contract',
+        },
+      },
+      metricsSpec: {
+        executableDripCount: {
```

Comment:
Swap this to a gauge metric - set it to zero if the drip is not executable, set to 1 if it is.